### ISSUE TITLE
Bug: item.tagName = 'no tag';

### ISSUE THREAD

**Issue by brianteeman** [paypal/skipto#77]
You can't have a space here or it kills the script.

To replicate it you can just try to open /skipto/example/skipTo-no-landmarks.html and then check your console

```
Uncaught DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('no tag') contains HTML space characters, which are not valid in tokens.
    at Object.renderMenuitemToGroup (http://localhost/skipto/compiled/js/skipto.js:384:30)
    at Object.renderMenuitemsToGroup (http://localhost/skipto/compiled/js/skipto.js:500:14)
    at Object.renderMenu (http://localhost/skipto/compiled/js/skipto.js:736:12)
    at Object.openPopup (http://localhost/skipto/compiled/js/skipto.js:859:12)
    at Object.handleButtonClick (http://localhost/skipto/compiled/js/skipto.js:917:14)
```

I am not a dev but I guess you could change it to no-tag 
https://github.com/paypal/skipto/blob/e0d218c7eb1c0c0be1f2ecfbe5792e6582323667/src/js/skipto.js#L489-L496

(I'm not a js dev but I'm good at finding issues)

**Comment by jongund** [comment]
@brianteeman 
This is fixed on pull #74

**Comment by brianteeman** [comment]
thanks
